viroconcom/read_write.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import csv
+
+
+def read_dataset(path='datasets/1year_dataset_A.txt'):
+    """
+    Reads a 2D dataset that uses a an ASCI format with ';' as a seperator.
+
+    This format has been used in the EC benchmark,
+    see https://github.com/ec-benchmark-organizers/ec-benchmark .
+
+    Parameters
+    ----------
+    path : string
+        Path to dataset including the file name, defaults
+        to '../datasets/1year_dataset_A.txt'
+    Returns
+    -------
+    x : ndarray of doubles
+        Observations of the environmental variable 1.
+    y : ndarray of doubles
+        Observations of the environmental variable 2.
+    x_label : str
+        Label of the environmantal variable 1.
+    y_label : str
+        Label of the environmental variable 2.
+    """
+
+    x = list()
+    y = list()
+    x_label = None
+    y_label = None
+    with open(path, newline='') as csv_file:
+        reader = csv.reader(csv_file, delimiter=';')
+        idx = 0
+        for row in reader:
+            if idx == 0:
+                x_label = row[1][1:] # Ignore first char (is a white space).
+                y_label = row[2][1:] # Ignore first char (is a white space).
+            if idx > 0: # Ignore the header
+                x.append(float(row[1]))
+                y.append(float(row[2]))
+            idx = idx + 1
+
+    x = np.asarray(x)
+    y = np.asarray(y)
+    return (x, y, x_label, y_label)
+
+
+def write_contour(x, y, path, label_x='Significant wave height (m)',

```

Comment:
I agree (mostly). Instead of angled brackets [] I will use normal brackets () as angled brackets in the SI standard mean "unit of" so you would write "the unit of mass if is kg" by writing [m] = kg.

To specify the units of a variable normal brackets should be used. This is also enforced by some academic journals.